spec/models/role_spec.rb

```diff
@@ -1,5 +1,54 @@
 require 'spec_helper'
 
-describe Role, :type => :model do
-  skip "add some examples to (or delete) #{__FILE__}"
+describe Role do
+  let(:person) { create(:person) }
+
+  describe 'validations' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.